mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/ElasticSearchIndexer.java

```diff
@@ -26,14 +26,34 @@
 import org.elasticsearch.action.bulk.BulkResponse;
 import org.elasticsearch.action.delete.DeleteResponse;
 import org.elasticsearch.action.index.IndexResponse;
-import org.elasticsearch.action.update.UpdateResponse;
 import org.elasticsearch.client.Client;
+import org.elasticsearch.common.Strings;
 import org.elasticsearch.index.query.QueryBuilder;
 
 import com.google.common.base.Preconditions;
 
 public class ElasticSearchIndexer {
 
+    public static class UpdatedDocumentPart {
```

Comment:
Its the representation of an update that the indexer should performed.

I just updated this inner class name